<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddImageSourceDlg</class>
 <widget class="QDialog" name="AddImageSourceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AddImageSourceDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background:#262626;
border:1px solid #585858;
}
#wgt_top{
background:#2F2F2F;
}
QLabel,QPushButton,QSlider{
background:transparent;
}
#button_confirm{
background-image: url(:/TQLive/images/btn_confirm.png);
background-repeat:no-repeat;
border:none;
}
#button_confirm:hover{
background-image: url(:/TQLive/images/btn_confirm_hover.png);
}
#button_confirm:pressed{
margin:1px;
padding:1px;
}
#button_cancel{
background-image: url(:/TQLive/images/btn_cancel.png);
background-repeat:no-repeat;
border:none;
}
#button_cancel:hover{
background-image: url(:/TQLive/images/btn_cancel_hover.png);
}
#button_cancel:pressed{
margin:1px;
padding:1px;
}
#label_opacity,#label_opacity_val{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;
background:transparent;
}
QSlider{
background:transparent;
}
#wgt_preview{
background:#3f3f3f;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <widget class="QWidget" name="wgt_top" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#button_close{
background:transparent;
background-image: url(:/TQLive/images/pb_close_pressed.png);
background-repeat:no-repeat;
border:none;
}
#button_close:hover{
background-image: url(:/TQLive/images/pb_close_normal.png);
}
#button_close:pressed{
margin:1px;
padding:1px;
}
QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#4AB134;
background:transparent;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>19</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_title">
        <property name="text">
         <string>添加图片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_close">
        <property name="minimumSize">
         <size>
          <width>12</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>12</width>
          <height>13</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="OBSQTDisplay" name="wgt_preview" native="true">
       <property name="minimumSize">
        <size>
         <width>480</width>
         <height>270</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>480</width>
         <height>270</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>19</number>
     </property>
     <property name="rightMargin">
      <number>19</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_opacity">
       <property name="text">
        <string>不透明度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_opacity">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_opacity_val">
       <property name="text">
        <string>50</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_confirm">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OBSQTDisplay</class>
   <extends>QWidget</extends>
   <header>qt-display.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
